src/components/Editor/Preview/Popup.js

```diff
@@ -34,6 +36,10 @@ type Props = {
   openLink: string => void
 };
 
+function isReactComponent(roots) {
+  return roots.some(root => root.name === "_reactInternalInstance");
```

Comment:
can we use https://reactjs.org/docs/react-api.html#isvalidelement ?

Reply:
Hmm it doesn't seem to work for our purposes. Something like:
`  const reactElem = <span>hi sam</span>;
  React.isValidElement(reactElem);`
yields `true`, but in Popup.js we're not dealing w/ react objects, we get `roots` which I'm not totally sure what is lol (its definitely an array of elements that do not look like react elements). I know that `roots.some(root => React.isValidElement(root))` yields false even when hovering on a react component